### ISSUE TITLE
WinForms Windows authentication - Callback javascript error (release 2.4.0)

### ISSUE THREAD

**Issue by AndrePinto-NET** [IdentityServer/IdentityServer4#3121]
I'm using an old WinForms application with Windows authentication together with Identity Server, and on Identity Server, I've now updated IdentityServer4.AspNetIdentity package from version 2.3.0 to 2.4.0 and IdentityServer4.EntityFramework from 2.3.2 to 2.4.0. From that moment on, I've got an error on ...connect/authorize/callback.... (only on the mentioned WinForms App, everything else works fine)

![image](https://user-images.githubusercontent.com/22496294/54988721-59759b80-4fb7-11e9-9e12-eb5de7fb1444.png)

When I downgraded the mentioned packages, everything worked again.

**Comment by chichiton** [comment]
I have the same problem in wpf webbrowser after update to 2.4

**Comment by martinw3** [comment]
Note this is likely to be the same issue I raised earlier in issue #3128 

Also something to be aware of if you want to disable this script error warnings (which won't fix the issue but might be something you want to do):

Disabling the scripts on the WebBrowser control using the ScriptErrorsSuppressed property doesn't do what you might think, as it as it silences ALL window popups not just the script errors ones. 

This was an issue for us as we required the client certificate prompt window to appear.

See the following resources:

https://blogs.msdn.microsoft.com/ieinternals/2011/05/04/the-web-browser-control-and-the-silent-flag/
https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.webbrowser.scripterrorssuppressed
  
The solution above in the MSDN article didn't work for us. The only solution I found that worked was the one listed here:

https://stackoverflow.com/a/17069105
